README.md

```diff
@@ -2,68 +2,101 @@
 
 TIMDEX Pipeline Lambdas is a collection of lambdas used in the TIMDEX Ingest Pipeline.
 
-## Format Handler
+## Format Input Handler
 
-Takes an input from EventBridge and formats the event data to work throughout the entire ingest pipeline.
+Takes input JSON (usually from EventBridge although it can be passed to a manual Step Function execution), and returns reformatted JSON matching the expected input data needed for the remaining steps in the TIMDEX pipeline Step Function.
 
-### Example Format input
+### Input fields
+
+#### Required fields
+
+- `harvest-type`: Must be one of `["full", "daily"]`. The provided harvest type is used in the input/output file naming scheme for all steps of the pipeline. It also determines the harvest logic as follows:
+  - `full`: Will perform a full harvest of all records from the provided `oai-pmh-host`.
```

Comment:
Optional since this is a personal preference but the `will` at the start of most of these descriptions seems unnecessary

Reply:
Agree, will change.